lms/static/js/spec/search/search_spec.js

```diff
@@ -572,18 +574,17 @@ define([
                         course_name: ''
                     }
                 }]
-            })]);
-            this.server.respond();
+            });
             expect($('.search-info')).toExist();
             expect($('.search-result-list')).toBeVisible();
             expect(this.$searchResults.find('li').length).toEqual(1);
         }
 
         function showsErrorMessage () {
```

Comment:
This function is not used anywhere. Deleting it gives me the same number of passed tests as it does with it.